src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss

```diff
@@ -49,12 +48,20 @@
   }
 
   button {
-    margin: 30px;
+    margin: 30px 0 30px 30px;
     font-family: var(--secondary-font);
   }
 
   h5 {
+    font-weight: bold;
     margin-left: 15px;
+    margin-bottom: 30px;
+    margin-top: 30px;
+  }
+
+  h3 {
+    font-weight: bold;
+    color: #000000;
```

Comment:
We use variables for colors, so try to find an appropriate one and use it here

Reply:
done